docs/extending/extending_form_builder.md

```diff
@@ -215,6 +215,17 @@ By adjusting them to your needs, you will change your email template.
 
 ### Configure sender details
 
-To send emails, you also need to configure `sender_address` in `config/packages/swiftmailer.yaml`.
-It acts as a sender and return address for all bounced messages.
-For details, see [Symfony Mailer Configuration Reference.](https://symfony.com/doc/5.0/reference/configuration/swiftmailer.html#sender-address)
+To send notification emails from the Form Builder, you need to configure `sender_address` in `config/packages/swiftmailer.yaml`. It acts as a sender and return address for all bounced messages. For more information, see [Symfony Mailer Configuration Reference.](https://symfony.com/doc/5.0/reference/configuration/swiftmailer.html#sender-address) Some providers require a `sender_address` to be defined, or outgoing mails will be bounced.
```

Comment:
Not all e-mail service providers are requiring the `sender_address` to be set - maybe we can change it to something like:
```suggestion
Some e-mail providers require the `sender_address` in `config/packages/swiftmailer.yaml` to be set. Therefore, to send notification emails from the Form Builder, you need to configure it . It acts as a sender and return address for all bounced messages. For more information, see [Symfony Mailer Configuration Reference.](https://symfony.com/doc/5.0/reference/configuration/swiftmailer.html#sender-address) Some providers require a `sender_address` to be defined, or outgoing mails will be bounced.
```